components/pkg-export-helm/src/chart.rs

```diff
@@ -47,7 +47,12 @@ impl<'a> Chart<'a> {
             .value_of("CHART")
             .unwrap_or(&manifest.pkg_ident.name)
             .to_string();
-        let version = matches.value_of("VERSION");
+        let pkg_version = manifest.pkg_ident.version.clone();
+        let version = matches.value_of("VERSION").or(
```

Comment:
Totally minor, but could you use `or_else` here instead, and only only clone a version when necessary?

Reply:
I first tried to keep it simple and not clone but not cloning meant that we'd have to borrow `manifest` and borrow it again before the first borrow is finished in this line:

```
    Ok(Self::new_for_manifest(manifest, name, chartfile, deps, ui))
```

Reply:
why prefer `or_else` btw? It only seems to make the call more complex.

Reply:
Actually we're moving the `manifest` while borrowed if we don't clone the version string:

```
error[E0505]: cannot move out of `manifest` because it is borrowed
  --> components/pkg-export-helm/src/chart.rs:59:35
   |
51 |             manifest.pkg_ident.version.as_ref().map(|s| {
   |             -------------------------- borrow of `manifest.pkg_ident.version` occurs here
...
59 |         Ok(Self::new_for_manifest(manifest, name, chartfile, deps, ui))
   |                                   ^^^^^^^^ move out of `manifest` occurs here

```